yoda/src/test/java/life/jukebox/AppTest.java

```diff
@@ -0,0 +1,38 @@
+package life.jukebox.yoda;
```

Comment:
Is this needed? I think you need to get rid of this, you most likely want to create a YodaGrpcServerTest but I'm not sure how trivial testing the rpc endpoint is (it'll most likely require [mocking](https://stackoverflow.com/questions/2665812/what-is-mocking) so that we're not making network calls during a test)